test/sql/003_binary_copy_test.sql

```diff
@@ -0,0 +1,14 @@
+-- bigistore binary_copy should copy data binary from country;
```

Comment:
Pretty sure this was a leftover of a copy/paste from our country extension:
https://github.com/adjust/pg-country/blob/433e1cddebfd04a30f3a866e0135798963844e47/spec/binary_copy_spec.rb#L8

It would be better to clean this up, not sure how to rephrase it though, "bigistore binary_copy should copy data binary from bigistore" doesn't really sound super clear.

Maybe:
```suggestion
-- bigistore binary_copy should copy data out and in again using binary format;
```

Reply:
Indeed! I'll fix it.